code/messaging/src/main/java/com/adobe/marketing/mobile/aepuitemplates/uimodels/AepDismissButton.kt

```diff
@@ -19,4 +19,4 @@ package com.adobe.marketing.mobile.aepuitemplates.uimodels
  * - `simple`: A simple dismiss icon.
  * - `circle`: A circular dismiss icon.
  */
-data class AepDismissButton(val style: String)
+data class AepDismissButton(val style: String? = null)
```

Comment:
This should be made generic to and IconButton before release.
Can please we add a TODO to ensure we do not miss this.